src/components/TimeCardPage/types.tsx

```diff
@@ -0,0 +1,18 @@
+export enum CellType {
+    Regular = "Regular", 
+    PTO = "PTO"
+}; 
+
+export enum CommentType { 
+    Comment = "Comment", 
+    Report = "Report", 
+}; 
+
+export const enum Review_Stages {
```

Comment:
Can these be not spaced lol

These are what I'm using rn : 

 * -- HoursSubmitted (Associate has submitted their hours worked)
 * -- HoursReviewed (Supervisor has reviewed and approved the associate-submitted hours)
 * -- ScheduleSubmitted (Supervisor has submitted the scheduled hours)
 * -- Finalized (Admin has approved the submitted hours and schedule, and resolved any issue necessary)

I'm basically using it like a list of statuses that have been completed

Reply:
Or I guess it's fine if they're spaced, we should just agree on what status we want

Reply:
We should agree of the string value, and then can you have an enum in javascript have multiple attributes, like a 'value' which is what the backend actually passes over and then a 'display value' where you can define what you want to be displayed to the user